app/scripts/templates/sign_in.mustache

```diff
@@ -19,6 +19,9 @@
     {{^error}}
       <div class="error"></div>
       <div class="success"></div>
+        {{#migration}}
+          <div class="info nudge">{{#t}}Already have a Firefox Account? Migrate your sync data by signing into it here.{{/t}}</div>
```

Comment:
Nit: Not sure what "it" refers to here. FxA? Sync?
Also, should that be "you sync" or "your Sync"?